aggsender/db/aggsender_db_storage.go

```diff
@@ -179,10 +211,38 @@ func getCertificateByHeight(db dbtypes.Querier,
 		"SELECT * FROM certificate_info WHERE height = $1;", height); err != nil {
 		return nil, getSelectQueryError(height, err)
 	}
-
 	return &certificateInfo, nil
 }
 
+// getCertificatesByHeight returns a certificates in the required table
+func getCertificatesByHeight(db dbtypes.Querier, table tableName,
```

Comment:
Consider making this a method, so then you can remove the `db` param and reuse the `db` reference from the instance. WDYT?
```suggestion
func (a *AggSenderSQLStorage) getCertificatesByHeight(table tableName,
```